docs/index.md

```diff
@@ -23,6 +23,7 @@ TestContainers makes it easy to launch useful Docker containers for the duration
  * [Apache Gora](https://gora.apache.org) - CouchDB testing
  * [Apache James](https://james.apache.org) - LDAP and Cassandra integration testing
  * [StreamSets](https://github.com/streamsets/datacollector) - LDAP, MySQL Vault, MongoDB, Redis integration testing
+ * [Playtika](http://www.playtika.com) - kafka, couchbase, mariadb, redis, neo4j, aerospike, memsql
```

Comment:
could you please fix the letter case? :) i.e. Kafka, MariaDB, MemSQL, etc :)

Reply:
:)
okay